handout/routing/routeroutlet.md

```diff
@@ -22,6 +22,6 @@ export class AppComponent {}
 
 In the above example, the component corresponding to the route specified will be placed after the `<router-outlet></router-outlet>` element when the link is clicked.
 
-[View Example](https://plnkr.co/edit/odsZq1pL43HtWcUr0taJ?p=preview)
+[View Example](https://plnkr.co/edit/OHfytJquXKm8jvSe2T9Y?p=preview)
```

Comment:
Ah you're right `plunker`'s previewer doesn't show route changes in the URL. Looked into just referencing a link that would just generate the full screen view but doesn't seem to work for some reason (http://run.plnkr.co/plunks/odsZq1pL43HtWcUr0taJ). 

There's a note right after this link that mentions you have to open it in full screen mode in order to see the route changes and that works just fine (the little blue icon top right underneath your username when you open the example). Do you think this is alright? Other examples in this chapter mention the same thing and unfortunately I don't think there's another way we can do this with `plunker`.